health-sdk/sdk/src/doc/source/flow.rst

```diff
@@ -1,58 +1,169 @@
-                                private val listener: ReviewFragmentListener
-    ) : FragmentFactory() {
-        override fun instantiate(classLoader: ClassLoader, className: String): Fragment {
-            return ReviewFragment(giniHealth, configuration, listener)
+    viewHolder.paymentComponentView.prepareForReuse()
+    viewHolder.paymentComponentView.isPayable = invoiceItem.isPayable
+    viewHolder.paymentComponentView.documentId = invoiceItem.documentId
+
+.. note::
+
+    The ``PaymentComponentView`` will only be visible if its ``isPayable`` property is ``true``.
+
+Show the MoreInformationFragment
+--------------------------------
+
+The ``MoreInformationFragment`` displays information and an FAQ about the payment feature.
+
+To instantiate it use ``MoreInformationFragment.newInstance()`` and pass in your ``PaymentComponent`` instance:
+
+.. code-block:: kotlin
+
+    MoreInformationFragment.newInstance(paymentComponent)
+
+Show the BankSelectionBottomSheet
+---------------------------------
+
+The ``BankSelectionBottomSheet`` displays a list of available banks for the user to choose from. If a banking app is not
+installed it will also display its Play Store link.
+
+To instantiate it use ``BankSelectionBottomSheet.newInstance()`` and pass in your ``PaymentComponent`` instance:
+
+.. code-block:: kotlin
+
+    BankSelectionBottomSheet.newInstance(paymentComponent)
+
+
+Show the ReviewFragment
+-----------------------
+
+The ``ReviewFragment`` displays an invoice's pages and extractions. It also lets users pay the invoice with the bank
+they selected in the ``BankSelectionBottomSheet``.
+
+To instantiate it use ``paymentComponent.getPaymentReviewFragment()`` and pass in the Gini Health API's document id of
+the invoice and the configuration for the screen. Also set a listener to get informed of events from the fragment:
+
+.. code-block:: kotlin
+
+    val reviewConfiguration = ReviewConfiguration(...)
+
+    val paymentReviewFragment = paymentComponent.getPaymentReviewFragment(
+        documentId, reviewConfiguration
+    )
+
+    paymentReviewFragment.listener = object : ReviewFragmentListener {
+        override fun onCloseReview() {
+            // Called only when the ``ReviewConfiguration.showCloseButton`` was set to ``true``.
+            // Dismiss the ReviewFragment.
+        }
+
+        override fun onNextClicked(paymentProviderName: String) {
+            // Log or track the used payment provider name.
+            // No action required, the payment process is handled by the Gini Health SDK.
         }
     }
 
-    supportFragmentManager.fragmentFactory = ReviewFragmentFactory(giniHealth)
+.. note::
 
+    ``paymentComponent.getPaymentReviewFragment()`` will load the document extractions asynchronously. It's a suspend
+    function and must be called from a coroutine. 
 
 ``ReviewFragment`` handles errors by default, displaying snackbars for errors, but it
```

Comment:
[Suggestion] Perhaps we should clarify that this default value is set in `ReviewConfiguration`. It's a bit unclear from the description what screen configurations `ReviewConfiguration` handles. Could we maybe add some details about the default values that it has? I may be going in too much details here, feel free to ignore this suggestion.